### PR TITLE
fix: product structured data

### PR DESCRIPTION
**Pull request by emersonlaurentino**
[Offer data in Structured Data is broken](https://validator.schema.org/#url=https%3A%2F%2Fstarter.vtex.app%2F4k-philips-monitor-99988213%2Fp).

So in this PR, I fixed it.

### CODE REVIEW

File: packages/core/src/pages/[slug]/p.tsx [hunk 1]
```diff
@@ -90,12 +106,7 @@ function Page({ product, sections, globalSections }: Props) {
         gtin={product.gtin}
         releaseDate={product.releaseDate}
         images={product.image.map((img) => img.url)} // Somehow, Google does not understand this valid Schema.org schema, so we need to do conversions
-        offersType="AggregateOffer"
```
Comment (emersonlaurentino) [inline packages/core/src/pages/[slug]/p.tsx:93]: `offersType` don't exists in the schema

**Comment by emersonlaurentino** [comment]
preview available at https://github.com/vtex-sites/starter.store/pull/105

File: packages/api/package.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
     "node": ">=12"
   },
   "scripts": {
-    "dev": "tsdx watch --entry ./local/server.ts --target node --onSuccess \"node ./dist\"",
-    "dev:no-server": "concurrently \"yarn generate -w\" \"tsdx watch\"",
+    "dev": "concurrently \"yarn generate -w\" \"tsdx watch\"",
+    "dev:server": "tsdx watch --entry ./local/server.ts --target node --onSuccess \"node ./dist\"",
```
Comment (icazevedo) [inline packages/api/package.json:18]: Leftover from the other PR I guess

File: packages/core/src/pages/[slug]/p.tsx [hunk 3]
```diff
@@ -51,6 +51,22 @@ function Page({ product, sections, globalSections }: Props) {
   const description = seo.description || storeConfig.seo.description
   const canonical = `${storeConfig.storeUrl}${seo.canonical}`
 
+  const getStructuredDataOffers = () => {
```
Comment (eduardoformiga) [inline packages/core/src/pages/[slug]/p.tsx:54]: do you think does it worth to move this logic to the `getStaticProps` and return the result as prop?

Comment (icazevedo) [reply packages/core/src/pages/[slug]/p.tsx:54]: +1

**Review by eduardoformiga — APPROVED** [review]
LGTM!

**Review by icazevedo — APPROVED** [review]

**Pull request merged**